blocks/videotext/videotext.css

```diff
@@ -0,0 +1,23 @@
+@media screen and (min-width: 951px) and (max-width: 3840px) {
+  .videotext>div {
+    display: grid;
+    grid-template-columns: 70% 30%;
+    padding-top: 10px;
+  }
+}
+
+@media only screen and (min-width: 375px) and (max-width: 950px) {
+  .videotext>div {
+    display: grid;
+    padding-top: 10px;
+  }
+}
+
+.videocontent video {
+  width: 100%;
+}
+
+.videotext .text {
+  font-size: var(--type-body-s-size);
+  padding-left: 10px;
+}
```

Comment:
Can you turn the CSS around so it is mobile first and use 600, 900 and 1200 breakpoints? See other blocks for examples:
1. default rules for mobile
2. override rules for tablet and desktop using media queries with `min-width:...`